### ISSUE TITLE
add wrap in anonymous function, wrap in arrow function, wrap in generator function, wrap in async function

### ISSUE THREAD

**Issue by capaj** [cmstead/js-refactor#30]
Showcase
```javascript
after(mock.stopAll);

// selecting mock.stopAll and using wrap in arrow function should produce
after(() => {
  mock.stopAll();
});

// selecting mock.stopAll and using wrap in anonymous function should produce
after(function () {
  mock.stopAll();
});

// selecting mock.stopAll and using wrap in generator function should produce
after(function *() {
  mock.stopAll();
});

// selecting mock.stopAll and using wrap in anonymous async function should produce
after(async function () {
  mock.stopAll();
});
```

Would love to be able to do that with your plugin.

**Comment by cmstead** [comment]
Sorry I missed this issue until now, I could definitely implement this without too much effort. : )

**Comment by cmstead** [comment]
I'm moving forward with arrow function, generator, and async function because simply omitting the name of the function in "Wrap in Function" will create an anonymous function.

**Comment by cmstead** [comment]
Released in v0.9.0
